scripts/simulate_alerts.py

```diff
@@ -19,6 +19,7 @@
 
 def main():
     GENERATE_DEDUPLICATIONS = True
+    SLEEP_INTERVAL = 0.2  # Configurable sleep interval
```

Comment:
Maybe make that an env variable with default value?

Reply:
possible I will get you back with those changes in new pull request